### PR TITLE
feat: add widgetbot chat widget

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

image

image

### CODE REVIEW

**Comment by ahmaxed** [comment]
it is good to go for desktop. the keyboard controls button could go to the left.
For the mobile, we should hide the widget button for now since it is overlapping with stream bitrate controls.

**Comment by naomi-lgbt** [comment]
Thanks for the feedback @ahmaxed - I've moved the keyboard controls on desktop to the left side of the screen. Rather than hiding the button on mobile, there was room to bring the encoding selector up higher so I went with that.

File: public/index.html [hunk 1]
```diff
@@ -101,4 +101,19 @@
...
   </body>
+  <script src="https://cdn.jsdelivr.net/npm/@widgetbot/crate@3" async defer>
+     new Crate({
+        server: '692816967895220344', // freeCodeCamp.org Official ᕕ(⌐■_■)ᕗ ♪♬
+        channel: '1254842489362317322' // #code-radio
+    })
+    const button = document.querySelector('crate > div').shadowRoot.querySelector("button");
+    button.style.bottom = "80px";
+    const embed = document.querySelector('crate > div').shadowRoot.querySelector(".embed");
+    embed.style.bottom = "80px";
```
Comment (ahmaxed) [inline public/index.html:112]:
```suggestion
    const button = document.querySelector('crate > div').shadowRoot.querySelector("button");
    button.style.bottom = "90px";
    const embed = document.querySelector('crate > div').shadowRoot.querySelector(".embed");
    embed.style.bottom = "90px";
```

**Comment by ahmaxed** [comment]
Thank you for the widget integration 🙏

**Pull request merged**